sdk/docs/BUILD.bazel

```diff
@@ -386,6 +386,7 @@ java_binary(
 daml_test(
     name = "ledger-api-daml-test",
     srcs = glob(["source/app-dev/code-snippets/**/*.daml"]),
+    additional_compiler_flags = ["--warn-bad-interface-instances=yes"],
```

Comment:
Add these flags because now the shake rules (which are used in a lot of integration tests) emit diagnostics, not just calls to `daml build`